src/pages/Dashboard/Dashboard.tsx

```diff
@@ -1,18 +1,28 @@
-import { PageSection, PageSectionVariants } from "@patternfly/react-core";
+import { PageSection, PageSectionVariants, Alert, Button } from "@patternfly/react-core";
 import React from "react";
-import { RouteComponentProps } from "react-router-dom";
+import { RouteComponentProps, useHistory } from "react-router-dom";
+import ArrowRightIcon from '@patternfly/react-icons/dist/js/icons/arrow-right-icon';
 import PastAnalysisTable from "../../components/pastAnalysis/PastAnalysisTable";
 import Wrapper from "../../containers/Layout/PageWrapper";
 
 type AllProps = RouteComponentProps;
 
 const DashboardPage: React.FC<AllProps> = () => {
 
+  const history = useHistory();
+
   return (
     <Wrapper>
       <PageSection className="page-body">
-        {/* <PageSection className="section-area" variant={PageSectionVariants.light}>
-        </PageSection> */}
+        <PageSection className="section-area" variant={PageSectionVariants.light}>
+          <Alert isInline variant="warning" title="Patient lookup designated to Create Analysis page">
+            <p>To look up a patient and create a new analysis, please go to the Create Analysis page.</p>
+          </Alert>
+          <Button variant="link" isLarge className="pf-u-pl-0" 
```

Comment:
Would this look better if I didn't remove left padding? 
(no left padding)
![image](https://user-images.githubusercontent.com/53355975/118150639-c55a2e80-b3e0-11eb-9d9e-aef9bc6a23cd.png)
vs
(default left padding)
![image](https://user-images.githubusercontent.com/53355975/118150745-e3279380-b3e0-11eb-9df8-043262b065b1.png)